com.github.akiraux.akira.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.akiraux.akira
+runtime: io.elementary.Platform
+runtime-version: '6'
+sdk: io.elementary.Sdk
+command: com.github.akiraux.akira
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # needed for perfers-color-scheme
+  - '--system-talk-name=org.freedesktop.Accounts'
+
+  - '--filesystem=home'
```

Comment:
Is this needed ?

Reply:
Yes, without this is not possible to save/load files from the home directory.

Reply:
We should switch to using Native dialogs wherever we can I guess

Reply:
Yeah, that's on my list.